styles/main.css

```diff
@@ -1,33 +1,23 @@
 
 body {
   background-color: powderblue;
-  font-family: "Times New Roman", Times, serif;
+  font-family: "Open Sans", sans-serif;
 }
 h1 {
   color: blue;
   background-image:url(../images/horizontal.jpg);
-width:100%;
-height:10%;
-top:0;
-bottom:0;
-right:0;
-left:0;
```

Comment:
These declarations are not necessary for this lab.  I am curious why you added them?